aws/nlb/nlb.tf

```diff
@@ -0,0 +1,291 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# VARIABLES / LOCALS / REMOTE STATE
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "vpc_id" {
+  type        = string
+  description = "VPC ID to associate with ALB"
+}
+variable "subnet_ids" {
+  type        = list(string)
+  description = "A list of subnet IDs to associate with ALB"
+}
+
+variable "internal" {
+  type        = bool
+  default     = false
+  description = "A bool flag to determine whether the ALB should be internal"
+}
+variable "http_redirect_enabled" {
```

Comment:
All these `https*` variables are not in use and should be removed. We will not have HTTPS with the NLB